graphics/Cargo.toml

```diff
@@ -23,6 +23,9 @@ log = "0.4"
 raw-window-handle = "0.5"
 thiserror = "1.0"
 bitflags = "1.2"
+cosmic-text = "0.9"
+rustc-hash = "1.1"
+unicode-segmentation = "1.6"
```

Comment:
I can't find where `unicode-segmentation` is used in this crate, maybe it just isn't used yet, pending the TODO with glyph vs graphmeme indices.

Reply:
Turns out it isn't necessary, since the glyphs that `cosmic-text` gives us already contain string indices representing the clusters. We can just find the right glyph: 956512338905bac0b156fdaf16fe3c3e07e97a84.